config/coordinating-skeleton-updates.md

```diff
@@ -0,0 +1,34 @@
+# Coordinating Skeleton Updates #
+
+When Lineage runs to push out skeleton updates to its descendent
+repositories, this often results in generating tens of PRs to review. To
+minimize this kraken-like behavior and reduce the burden on team of so many
+PR reviews, we try to coordinate updates to the skeletons in batches.
+
+## Checking the Lineage GitHub Action ##
+
+First, check the status of the [lineage GitHub Action]. If it's about to run,
```

Comment:
Fix this broken link.

Reply:
The link isn't actually broken - there's a section at the bottom that has links for re-use through the document. I'll move it upwards so it's more-obvious though.

Reply:
Resolved in https://github.com/cisagov/development-guide/pull/32/commits/70d0524f12a1dbc90ee23079b0167b62377a74ad

Reply:
Ah, I had forgotten about that feature of Markdown!